amlb/datasets/file.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, cache_dir=None):
         self._cache_dir = cache_dir if cache_dir else tempfile.mkdtemp(prefix='amlb_cache')
 
     @profile(logger=log)
-    def load(self, dataset, fold=0):
+    def load(self, dataset, fold=0, timestamp_column=None):
```

Comment:
you obtained this new column using
```
timestamp_column=self._task_def.dataset['timestamp_column']
```
so you already have the information in the `dataset` object